### PR TITLE
[Editor & ModManager] 1.0.6.3 Remember Profile bug

### PR DESCRIPTION
**Pull request by shoushou1106**
When the default selected game is removed, an exception will occur start from the second launch

Reproduction method:
1. start frosty and make sure `Remember Profile Choice` is enabled  
close frosty
2. remove default game (just need to move game.exe away)  
3. start frosty  
close frosty
(Profile will be removed now)
4. start frosty (exception here)
(Profile is removed and cannot be found)
<details><summary>Exception</summary>
<p>

```
The given key was not present in the dictionary.

   at System.ThrowHelper.ThrowKeyNotFoundException()
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Frosty.Core.Config.get_Item(String option, ConfigScope scope, String profile) in Path-to-repo:\FrostyToolsuite\SuperCheatedFrostyToolsuite\FrostyPlugin\Config.cs:line 242
   at Frosty.Core.Config.Get[T](String option, T defaultValue, ConfigScope scope, String profile) in Path-to-repo:\FrostyToolsuite\SuperCheatedFrostyToolsuite\FrostyPlugin\Config.cs:line 215
   at Frosty.Core.FrostyConfiguration..ctor(String profile) in Path-to-repo:\FrostyToolsuite\SuperCheatedFrostyToolsuite\FrostyPlugin\FrostyConfiguration.cs:line 25
   at FrostyModManager.App.Application_Startup(Object sender, StartupEventArgs e) in Path-to-repo:\FrostyToolsuite\SuperCheatedFrostyToolsuite\FrostyModManager\App.xaml.cs:line 130
   at System.Windows.Application.OnStartup(StartupEventArgs e)
   at System.Windows.Application.<.ctor>b__1_0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
``` 

</p>
</details> 

This bug can be solved manually by deleting `"DefaultProfile"` in config.json

### CODE REVIEW

File: FrostyModManager/App.xaml.cs [hunk 1]
```diff
@@ -129,18 +129,26 @@ private void Application_Startup(object sender, StartupEventArgs e)
...
                         Config.Save();
                     }
                 }
-                else
-                {
-                    Config.Add("UseDefaultProfile", false);
-                    Config.Save();
-                }
-            }
+//                else
```
Comment (wannkunstbeikor) [inline FrostyModManager/App.xaml.cs]: does this cause any issues or why is it commented out?

Comment (shoushou1106) [reply FrostyModManager/App.xaml.cs]: After the error occurs, user may not want to disable it. If thats wrong u can uncomment it

Comment (wannkunstbeikor) [reply FrostyModManager/App.xaml.cs]: no it makes sense, just remove it then completely

**Comment by wannkunstbeikor** [comment]
can u make a pr to 1.0.6.4

**Review by wannkunstbeikor — APPROVED** [review]

**Pull request closed without merging**